### PR TITLE
10.0 mig auditlog

### PR DESCRIPTION
**Pull request by BT-fgarbely**

### CODE REVIEW

**Review by sebalix — COMMENTED** [review]
@BT-fgarbely Good job! You even updated screenshots :tada: 
Tested on runbot, it works fine (fast/full log, subscribe/unsubscribe rules...).
Can you squash all OCA Transbot commits into one?

**Comment by BT-fgarbely** [comment]
Hello @sebalix 

Thanks a lot for your feedback.

I don't know how to squash the commits.
I get the commits from the Branch 9.0 according to the migration guide:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#technical-method-to-migrate-a-module-from-90-to-100-branch

Regards

**Comment by LoisRForgeFlow** [comment]
Hope this helps you: https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests @BT-fgarbely

File: README.md
Comment (lasley) [inline README.md]: Please revert this change - README.md is managed by a bot, so this change will create some conflicts.

File: auditlog/__manifest__.py
Comment (lasley) [inline auditlog/__manifest__.py]: Remove empty key

File: auditlog/data/ir_cron.xml
Comment (lasley) [inline auditlog/data/ir_cron.xml]: `<odoo noupdate="1">` & can remove the data tag

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @BT-fgarbely - few comments inline, but otherwise looks good.

Note on the squashing - please give it a shot first using [the instructions in the wiki](https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests), because it is definitely a skill that is handy as a developer. If all else fails though, I will squash them for you before merge.

**Comment by BT-fgarbely** [comment]
Hello @lasley and @lreficent  

Thanks for your hints.
I changed the code according to your review and I tried to squash the Transbot commits.
It did something, but I don't know if it worked as expected.

Please can you have a look again?

Thanks a lot.

**Comment by lasley** [comment]
Hey @BT-fgarbely - Hrmmm looks like you merged the squashed items instead of force pushing them in. We're trying to rewrite timeline, so we don't want the merge.

No matter though, I'll handle for you. I recommend playing around with this a bit though, you will find that things end up a lot more organized afterwards. It also has the added benefit of making you look like you write perfect code, because there are no commits indicating PEP fixes, PR reviews, or troubleshooting rabbit holes you ended in.

**Comment by lasley** [comment]
@BT-fgarbely - please check the box to allow maintainers to edit your branch so that I may assist - https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by BT-fgarbely** [comment]
Hello @lasley 

I checked the "Allow edits from maintainers." checkbox.

Regards

**Comment by lasley** [comment]
Hrmmm I'm still getting a permissions error when pushing the squash. Not sure what's wrong if the Allow edits from maintainers is enabled 😕

**Comment by BT-fgarbely** [comment]
Hello @lasley 

This is really strange.
The flag "Allow edits from maintainers." in this pull request is checked.
auswahl_002


My fork is not created from the OCA repository. Our company has a fork from the OCA repository, and my fork comes from our company fork.
Maybe this could be a problem? I don't know. But in my understanding, this should be no problem.

Another question.
I saw that there is a pull request from @sebalix into my fork (https://github.com/BT-fgarbely/server-tools/pull/1). Should I merge this changes into my branch?
auswahl_001


Regards,

**Comment by sebalix** [comment]
Hi,
I prefer to close the PR you mention, it was to port a fix from @StefanRijnhart but I also squash some commits, but I think I was doing wrong with this last point...

**Comment by lasley** [comment]
@BT-fgarbely Hrmm the check box is supposed to allow me to push to your fork, which is what I was trying. I'll give it another shot today, otherwise will give you some alternate instructions to just switch over to mine - there are many ways to skin a cat.

@sebalix Yeah the commit squashing will just add more commits into the mix here. What about the fix though, was it actually not valid?

**Comment by sebalix** [comment]
@lasley no it is still valid, but we can port it after this PR is merged, this is just a cherrypick. No need to add extra work on this one.

**Comment by lasley** [comment]
Alright sweet - I still can't push to the fork here though. I'm actually kind of curious why & think we may be hitting a bug with Github's new feature, so I contacted them.

Side note - we just started using this module in vertical-medical the other day & functional tests are also good 👍

**Comment by lasley** [comment]
Hah well a test edit went through, but force push still isn't working. I'm working with Github support though, seems we may have hit some sort of edge case that now intrigues the hell out of me.

Will provide update soon!

**Comment by lasley** [comment]
@BT-fgarbely Github engineering has confirmed that this is an issue with the fork from a fork. They're working on it, but that's obviously going to take a while.

We can do one of two things here. I can either supersede this PR with [my branch](https://github.com/laslabs/server-tools/tree/10.0-mig-auditlog), or you can just add the `LasLabs` fork as a remote then do the following:

```
git remote update
git checkout 10.0-mig-auditlog
git reset --hard 7b8bd4d
git push -f
```

**Comment by BT-fgarbely** [comment]
Hello @lasley 

I added the LasLabs fork as remote and executed your suggested commands.

Regards

**Review by lasley — APPROVED** [review]
Thanks @BT-fgarbely

**Review by sebalix — APPROVED** [review]
Cool, thank you @BT-fgarbely and @lasley . Tested on runbot once again, :+1:

**Comment by lasley** [comment]
Merging, this has been approved for long enough - we were just battling with git 😆

**Comment by BT-fgarbely** [comment]
Hello @lasley and @sebalix 

Thanks a lot for your help and your support on this issue.

Regards

**Pull request merged**
